google/cloud/gkehub_v1/configmanagement_v1/types/configmanagement.py

```diff
@@ -189,10 +189,12 @@ class PolicyController(proto.Message):
         template_library_installed (bool):
             Installs the default template library along
             with Policy Controller.
+            This field is a member of `oneof`_ ``_template_library_installed``.
         audit_interval_seconds (int):
             Sets the interval for Policy Controller Audit
             Scans (in seconds). When set to 0, this disables
             audit functionality altogether.
+            This field is a member of `oneof`_ ``_audit_interval_seconds``.
```

Comment:
@busunkim96 , The `oneof` target link is missing in the docstring for `PolicyController`. Should this field include the `oneof` reference?

Reply:
Huh that's odd, the intent was that the link ref always gets added if there's a oneof in the message:

https://github.com/googleapis/gapic-generator-python/blob/a0e25c8c00391b99a351e667eddc8b4fecad30d8/gapic/templates/%25namespace/%25name_%25version/%25sub/types/_message.py.j2#L5-L16

I'm in favor of opening a bug and proceeding to merge this PR. The link is not critical for this message since both oneofs only have a single member.

Reply:
Filed https://github.com/googleapis/gapic-generator-python/issues/1083